src/components/AccountDrawer/MiniPortfolio/index.tsx

```diff
@@ -24,6 +25,7 @@ const Wrapper = styled(Column)`
   flex-direction: column;
   height: 100%;
   gap: 12px;
+  margin-bottom: ${isMobile ? '48px' : '0px'};
```

Comment:
instead of using a hook you can use the built in css styling. You can replace this with 
```
@media screen and (max-width: ${BREAKPOINTS.sm}px) {
    margin-bottom: 48px;
  }

Reply:
Done!